src/PresentationalComponents/SystemsTable/SystemsTable.css.map

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Hm. not sure if the source map should be committed. I've never seen it anywhere else and I think most are git-ignoring it.

Reply:
I just had 'watch sass' on so when I save in the sass file it automatically compiles these. When updating the sass files wouldnt these change in master regardless?